lib/datasets/mnist.rb

```diff
@@ -0,0 +1,106 @@
+require 'zlib'
+
+require_relative "dataset"
+
+module Datasets
+  class MNIST < Dataset
+    module Pixelable
+      def pixels
+        data.unpack("C*")
+      end
+
+      def to_h
+        hash = super
+        hash[:pixels] = pixels
+        hash
+      end
+    end
+
+    class Record < Struct.new(:data, :label)
+      include Pixelable
```

Comment:
We don't need to create `Pixelable` module like CIFAR dataset.
Because there is only one record class for MNIST dataset.

Reply:
I understood your comment.
I removed `Pixelable` module.